.travis.yml

```diff
@@ -4,7 +4,3 @@ perl6:
 install:
   - rakudobrew build-panda
   - panda --notests installdeps .
-script:
```

Comment:
This `script:` doesn't run well, and [default test script of Travis Perl6 support](https://docs.travis-ci.com/user/languages/perl6#Default-Test-Script) seems to be enough to run tests, so I removed this `script:`.
If the (old) `script` is still necessary, I can revert these changes.